examples/pytorch/nlp/huggingface_models/language-modeling/quantization/auto_round/llama3/README.md

```diff
@@ -0,0 +1,197 @@
+
+```bash 
+# neural-compressor-pt
+INC_PT_ONLY=1 pip install git+https://github.com/intel/neural-compressor.git@master
+# auto-round
+pip install git+https://github.com/intel/auto-round.git@more-ar-ext
+# other requirements
+pip install -r requirements.txt
+```
+
+
+## Quantization
+
+### Demo (`MXFP4`, `MXFP8`, `NVFP4`, `uNVFP4`)
+
+```bash
+CUDA_VISIBLE_DEVICES=0 python quantize.py  \
+    --model_name_or_path facebook/opt-125m  \
+    --quantize \
+    --dtype MXFP8 \
+    --enable_torch_compile \
+    --low_gpu_mem_usage \
+    --export_format auto_round \
+    --export_path OPT-125M-MXFP8 \
+    --accuracy \
+    --tasks lambada_openai \
+    --eval_batch_size 8
+```
+
+Notes:
+- Use `--export_format auto_round` for `MXFP4`, `MXFP8` data type and do inference as below.
+- Use `--export_format llm_compressor` for `NVFP4` data type since public vLLM supports it.
+- Use `--export_format fake` for `uNVFP4` data type since it's not fully supported.
+- Setting `--quant_lm_head` applies `--dtype` for the lm_head layer.
+- Setting `--iters 0` skips AutoRound tuning and uses RTN method.
+
+
+#### Target_bits
+
+To achieve optimal compression ratios in mixed-precision quantization, we provide the `--target_bits` argument for automated precision configuration.
+
+- If you pass a single float number, it will automatically generate an optimal quantization recipe to achieve that target average bit-width.
+- If you pass multiple float numbers, it will generate multiple recipes for different target bit-widths, allowing you to compare trade-offs between model size and accuracy.
+
+Example usage:
+
+```bash
+CUDA_VISIBLE_DEVICES=0 python quantize.py  \
+    --model_name_or_path facebook/opt-125m \
+    --quantize \
+    --dtype MXFP4 \
+    --target_bits 6.5 7 7.3 \
+    --tune_limit 100 \
+    --enable_torch_compile \
+    --low_gpu_mem_usage \
+    --export_format auto_round \
+    --export_path OPT-125m-MXFP4-MXFP8 \
+    --accuracy \
+    --tasks lambada_openai \
+    --eval_batch_size 8
+```
+
+
+### Llama3 Quantization Recipes
+
+#### Llama 3.1 8B MXFP8
+
+AutoRound helps improve the accuracy, `iters` and `nsamples` is higher than default.
+```bash
+# Quantize and export AutoRound format
+CUDA_VISIBLE_DEVICES=0 bash run_quant.sh --topology=Llama-3.1-8B --dtype=mxfp8 --input_model=/models/Meta-Llama-3.1-8B-Instruct --output_model=Llama-3.1-8B-MXFP8
+```
+
+#### Llama 3.1 8B MXFP4 (Mixed with MXFP8, Target_bits=7.8)
+
+```bash
+CUDA_VISIBLE_DEVICES=0 bash run_quant.sh --topology=Llama-3.1-8B --dtype=mxfp4_mixed --input_model=/models/Meta-Llama-3.1-8B-Instruct --output_model=Llama-3.1-8B-MXFP4-MXFP8
+```
+
+#### Llama 3.3 70B MXFP8
+
+```bash
+CUDA_VISIBLE_DEVICES=0 bash run_quant.sh --topology=Llama-3.3-70B --dtype=mxfp8 --input_model=/models/Llama-3.3-70B-Instruct/ --output_model=Llama-3.3-70B-MXFP8
+```
+
+#### Llama 3.3 70B MXFP4 (Mixed with MXFP8, Target_bits=5.8)
```

Comment:
besides giving exact target bits (here 5.8), is it possible to add an example cml to generate mixed bits model with INC autotune feature, for a new model, user may be able to get a good model throught autotune without trying different target-bits?